### PR TITLE
Only test for incremental streams in standard incremental test

### PR DESCRIPTION
**Pull request by sherifnada**
## What
The incremental standard test asserts that a sync run with the state output from the previous sync outputs no records. For this check to make sense, we need to only sync incremental streams. This PR filters the catalog to do so.

### CODE REVIEW

File: airbyte-integrations/bases/standard-source-test/build.gradle [hunk 1]
```diff
@@ -3,10 +3,6 @@ plugins {
     id 'airbyte-docker'
 }
 
-application {
```
Comment (jrhizor) [inline airbyte-integrations/bases/standard-source-test/build.gradle:6]: Why is this removed?

Comment (sherifnada) [reply airbyte-integrations/bases/standard-source-test/build.gradle:6]: it's duplicated -- see below in the file

**Review by cgardens — APPROVED** [review]
how do the standard tests handle the case if a source does NOT support full refresh (it is incremental only), right now? is that something we need to build support for now?

**Comment by sherifnada** [comment]
yes, will add in a coming PR

**Review by jrhizor — APPROVED** [review]
